engine/Shopware/Models/Document/Document.php

```diff
@@ -154,6 +164,30 @@ public function getName()
         return $this->name;
     }
 
+    /**
+     * Sets the document's key
+     *
+     * @param string
+     *
+     * @return \Shopware\Models\Document\Document
+     */
+    public function setKey($key)
+    {
+        $this->key = $key;
+
+        return $this;
+    }
+
+    /**
```

Comment:
"unique key"

Reply:
Fixed, comment not updated because it is anchored in the line before. 😉